docs/generated/index.html

```diff
@@ -195,7 +195,7 @@ <h1>OJP - <strong>O</strong>pen API for distributed <strong>J</strong>ourney <st
 			<!--======= row--><tr><td class="tableblock halign-left valign-top" colspan="2"><p class="tableblock"><code>TrainNumber</code></p></td><td class="tableblock halign-left valign-top"><p class="tableblock"><span title="optional, single">0:1</span></p></td><td class="tableblock halign-left valign-top"><p class="tableblock"><em>xs:normalizedString</em></p></td><td class="tableblock halign-left valign-top"><p class="tableblock">Contains the TrainNumber from NeTEx (TRAIN NUMBER from Transmodel). If several TrainNumber types exist, use the commercial one. In some cases also non-train modes will use TrainNumber.</p></td></tr>
 			<!--======= row--><tr><td class="tableblock halign-left valign-top" colspan="2"><p class="tableblock"><code>→‍siri:VehicleRef</code></p></td><td class="tableblock halign-left valign-top"><p class="tableblock"><span title="optional, single">0:1</span></p></td><td class="tableblock halign-left valign-top"><p class="tableblock"><em>→‍siri:VehicleRef</em></p></td><td class="tableblock halign-left valign-top"><p class="tableblock">Contains the Vehicle reference of the vehicle. In Transmodel this may be the VEHICLE Id.</p></td></tr>
-			<!--======= row--><tr><td class="tableblock halign-left valign-top" colspan="2"><p class="tableblock"><code>→‍<a href="#OperatorRef" title="OperatorRef">OperatorRef</a></code></p></td><td class="tableblock halign-left valign-top"><p class="tableblock"><span title="optional, single">0:1</span></p></td><td class="tableblock halign-left valign-top"><p class="tableblock"><em>→‍<a href="#OperatorRef" title="OperatorRef">OperatorRef</a></em></p></td><td class="tableblock halign-left valign-top"><p class="tableblock">Reference to an Operator ([TMv6] company  providing public transport services.) Reference to transport operatorIdentifier of the operator of the sharing service </p></td></tr>
+			<!--======= row--><tr><td class="tableblock halign-left valign-top" colspan="2"><p class="tableblock"><code>OperatorRefs</code></p></td><td class="tableblock halign-left valign-top"><p class="tableblock"><span title="optional, single">0:1</span></p></td><td class="tableblock halign-left valign-top"><p class="tableblock"><em>+<a href="#OperatorsRefsStructure" title="OperatorsRefsStructure">OperatorsRefs</a></em></p></td><td class="tableblock halign-left valign-top"><p class="tableblock">Reference to the OPERATORS. Multiple OPERATORS in case a ContinuousLeg can be operated by more than one operator, especially in the ALTERNATIVE MODE OF OPERATION where there can be dozens of taxi companies and many free floating sharing companies.</p></td></tr>
```

Comment:
Not sure if there is a sign too much:  "<em>+<a href="#OperatorsRefsStructure"..."
The plus "+" seems not required here too me.

Reply:
@AndreasAtSBB , the `+` and `-` signs are git syntax for added / removed lines.
Please do not concentrate too much on diffs of html, those are automatically rendered.
Have a look at the changes in the XSD and use https://htmlpreview.github.io/ to have a look at the rendered html. Using this link does not work in all parts though, the permallink indicators are shown there twice.
This might be done by https://github.com/VDVde/OJP/issues/288

Also, when reviewing, please start a review instead of using "single comment" because otherwise an email notification will be sent for each single comment.